pkg/ring/model.go

```diff
@@ -751,7 +751,12 @@ func (d *Desc) FindDifference(o codec.MultiKey) (interface{}, []string, error) {
 			if !tokensEqual(ing.Tokens, oing.Tokens) {
 				tokensChanged = true
 			}
```

Comment:
We need to consider now that the tokens might not change if the timestamp is in the past. We should mark as tokensChanged only if we add the instance to toUpdate

Reply:
right, I agree. That will save us some resolveconflict executions. I'll make the change.